frontend/apps/reader/modules/readerfont.lua

```diff
@@ -209,6 +209,7 @@ function ReaderFont:onSetFontSize(new_size)
         timeout = 2,
     })
 
+    --- @note: Shouldn't we let this propagate, to allow stuff to react to font size changes?
```

Comment:
I guess nothing else _should_ really have to care about this module "private" event.